### DISCUSSION TITLE
In ios app unable to edit My Work labels. An error occurs on saving

### DISCUSSION THREAD

**Discussion by stasgm** [community/community#58736] [Mobile] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/8355925/43351aa3-0f45-4558-8138-3c849e21fcd9)

**Comment by esahmed21** [comment]
Hi @stasgm  👋

Thank you for reaching out! 

I'm trying to reproduce it but didn't happen with me, would you mind sharing the app version you're using currently and if there are specific steps to follow to get this error 🙌?

Also if you're not using the latest update, would you mind update the app, and try again and let me know if this fix the issue?

In all cases This is definitely a problem that should be fixed, I'll create issue internally and investigate it with the team.

**Comment by stasgm** [comment]
Hi @esahmed21
You can see the version number and my steps below.

![image](https://github.com/community/community/assets/8355925/f45ee511-5b33-4b09-9e51-595bffd81187)

https://github.com/community/community/assets/8355925/f194929b-d034-473c-b20f-e30212cead3a

**Reply by esahmed21** [reply, marked as answer]
@stasgm  Thank you so much for providing this information 🙌 we have submitted a fix already that will be coming public very soon!

Thanks again for your patience 🙏

**Reply by stasgm** [reply]
I checked, it's fixed in the last patch. Thank you
